PWGCF/JCorran/Core/JFFlucAnalysis.cxx

```diff
@@ -352,22 +127,8 @@ TComplex JFFlucAnalysis::Four(int n1, int n2, int n3, int n4)
 #undef C
 
 //________________________________________________________________________
-void JFFlucAnalysis::UserExec(Option_t* /*popt*/)
+void JFFlucAnalysis::UserExec(Option_t* popt)
```

Comment:
If you are not using `popt` don't leave it available, this is an unused variable warning which is going to be converted in an error in the coming days

Reply:
Ok, will remove it next time. There was some issue with the cpplint, but probably because of the comment. Thanks!